### PR TITLE
feat(lending): add swapper quote error handling

### PR DESCRIPTION
**Pull request by chouandy**

### CODE REVIEW

File: packages/lending/src/adapter.ts [hunk 1]
```diff
@@ -534,55 +556,55 @@ export class Adapter extends common.Web3Toolkit {
             // 1-5-1. the supply amount is the swap quotation output
             supplyInput = swapQuotation.output;
```
Comment (Jeff-CCH) [inline packages/lending/src/adapter.ts:562]: Wondering if we need to add a try-catch for Ln.555

Comment (chouandy) [reply packages/lending/src/adapter.ts:562]: Fixed~

**Comment by Jeff-CCH** [comment]
LGTM

**Comment by LuPoYi** [comment]
LGTM

**Review by LuPoYi — COMMENTED** [review]
LGTM

**Pull request merged**